employee-rostering-frontend/src/ui/header/Navigation.tsx

```diff
@@ -14,17 +14,18 @@
  * limitations under the License.
  */
 
-import { Nav, NavItem, NavList, NavVariants } from '@patternfly/react-core';
+import { Nav, NavItem, NavList } from '@patternfly/react-core';
 import * as React from 'react';
 import { useTranslation } from 'react-i18next';
 import { RouteComponentProps } from 'react-router';
 import { Link, withRouter } from 'react-router-dom';
 
-export const Navigation = ({ location }: RouteComponentProps) => {
+export const Navigation = ({ variant, location }: RouteComponentProps & { variant: "default" | "horizontal"}) => {
```

Comment:
Could you
```ts
export type NavigationProps = RouteComponentProps & { variant: 'default' | 'horizontal' };
```
and use `NavigationProps` in the test to avoid repeating code?